### PR TITLE
Get start time API

### LINKED ISSUE #9: GET: `/api/punch-in-time`
## API description
- URL parameters: None
- Authen header: Required.
- Body: None
- Description: This API will return the user's `TimeEntry`'s `startTime` for today. If it's `null`, return null. Otherwise, return time in the format: `HH:MM:SS`. Access token is used to get the user. One edge case is there is no `TimeEntry` created for today, then simply return null.

### PR DESCRIPTION
**Pull request by CL-17656**
This should work but I have not been able to test it in swagger as it keeps on saying anonymous objects do not have a account_set field

### CODE REVIEW

File: account/views.py [hunk 1]
```diff
@@ -40,3 +40,28 @@ def post(self, request):
...
+                                status=status.HTTP_200_OK)
+            else:
+                entry = time_entries.first()
+                if entry.start_time == None:
+                    return Response({'status': 'success', 'Response': {'start time': None}}, 
+                                status=status.HTTP_200_OK)
+                else:
+                    time = entry.start_time
+                    return Response({'status': 'success', 'Response': {'start time': time}}, 
+                                status=status.HTTP_200_OK)
```
Comment (nguyenvothuan) [inline account/views.py:63]: Nice job handling these cases!

File: account/views.py [hunk 2]
```diff
@@ -40,3 +40,28 @@ def post(self, request):
         except Exception as e:
             return Response({'status': False, 'message': str(e)},
                             status=status.HTTP_400_BAD_REQUEST)
+
+class GetStartTime(APIView):
+    def get(self, request):
+        try:
+            user = request.user
```
Comment (nguyenvothuan) [inline account/views.py:47]: so the reason you are getting Anonymous User maybe you didn't login correctly. Check my loom video on the previous PR for how to login with an access token.

File: account/urls.py [hunk 3]
```diff
@@ -13,9 +13,11 @@
 
 # local imports.
 from account.views import Punch
+from account.views  import GetStartTime
 
 app_name = 'accounts'
 
 urlpatterns = [
     path('punch', Punch.as_view(), name='punch in punch out'), 
+    path('GetStartTime', GetStartTime.as_view(), name = "get start time")
```
Comment (nguyenvothuan) [inline account/urls.py]: this should be the name of the API, check your issue for API description and make sure the pathname here matches that description.

File: account/urls.py [hunk 4]
```diff
@@ -13,9 +13,11 @@
 
 # local imports.
 from account.views import Punch
+from account.views  import GetStartTime
```
Comment (nguyenvothuan) [inline account/urls.py]: you can do 2 imports Punch and GetStartTime in one line

**Review by nguyenvothuan — CHANGES_REQUESTED** [review]

**Comment by nguyenvothuan** [comment]
Can you record a Loom video for a demo? Can be really helpful for others to see how your API will work.

**Comment by CL-17656** [comment]
Video: https://share.vidyard.com/watch/UokNwGCscPiQtmdSNpX8cL?

**Comment by nguyenvothuan** [comment]
LGTM!

**Review by nguyenvothuan — APPROVED** [review]
LGTM
